static/sass/_pattern_p-card.scss

```diff
@@ -51,12 +51,13 @@
     @extend %vf-has-box-shadow;
 
     background-color: $color-x-light;
-    border-radius: $spv-inner--x-small;
+    border-radius: 0.125rem;
     color: $color-dark;
     display: grid;
     grid-template-rows: 9.5rem auto 2.25rem;
```

Comment:
that's a lot of hard-coded values. Isn't it possible to make the tallest card determine height of all cards on that row?

Reply:
There were a lot of requirements to make the cards behave in a very specific way, and this was the best solution I could come up with at the time. I think now some of those requirements have changed, and when we implement the badges we should also look at the cards and make sure we use something more vanilla @danielmutis .

Reply:
sounds good, thanks!